src/routes/v2/authenticated/sites.ts

```diff
@@ -141,7 +182,18 @@ export class SitesRouter {
       this.authorizationMiddleware.verifySiteMember,
       attachReadRouteHandlerWrapper(this.getSiteInfo)
     )
-
+    router.get(
+      "/:siteName/launchInfo",
+      attachSiteHandler,
+      this.authorizationMiddleware.verifySiteMember,
+      attachReadRouteHandlerWrapper(this.getSiteLaunchInfo)
+    )
+    router.post(
+      "/:siteName/launchSite",
```

Comment:
yea routes lets use `-` for separator

Reply:
hmm I think maintaining consistency here might be more beneficial. Since the rest of the authenticated sites need to use `camelCasing` and changing things at the API level can increase the surface area for bugs, am concerned about making this change at all (even if we agree I think this should be tacked in a separate PR ya). Thoughts?

Reply:
if the rest of the routes in this router use `camelCase` then ok. otherwise, still prefer kebab.